docs/components/page-title.md

```diff
@@ -19,7 +19,7 @@ eleventyNavigation:
 - If a page needs more than one button, use a [Bootstrap Dropdown button](https://getbootstrap.com/docs/5.3/components/dropdowns/#single-button).
 - Remove the [Badges](/components/badges/) section if you don’t need them.
 - Try to keep to just two or three Badges.
-- Most badges in the Page Title, except Warning and Danger, will be dimmed to avoid undue emphasis.
+- [Badges](/components/badges/) are no longer dim by default in the Page Title to allow for flexibility.
```

Comment:
Do we want to make this a default change as this would be breaking? If so we should call it out and how to restore that functionality.

Reply:
Added instruction for restoring the dim badges here too